### DISCUSSION TITLE
Can it be achieved, is the Visual C++ runtime free to install?

### DISCUSSION THREAD

**Discussion by shouhujishu** [cefsharp/CefSharp#4302] [Q&A]
Comes with Visual C++ runtime file implementation, without installing Visual C++ to run cefsharp105.xx? Can it be implemented?

**Comment by amaitland** [comment]
Visual C++ is freely available from Microsoft. 

Please read https://github.com/cefsharp/CefSharp/wiki/Frequently-asked-questions#Including_vcredist
